pkg/apis/kops/v1alpha1/zz_generated.conversion.go

```diff
@@ -1250,6 +1256,8 @@ func Convert_kops_KubeSchedulerConfig_To_v1alpha1_KubeSchedulerConfig(in *kops.K
 
 func autoConvert_v1alpha1_KubeletConfigSpec_To_kops_KubeletConfigSpec(in *KubeletConfigSpec, out *kops.KubeletConfigSpec, s conversion.Scope) error {
 	out.APIServers = in.APIServers
+	out.KubeconfigPath = in.KubeconfigPath
```

Comment:
No... I just run "make apimachinery" to update those auto-generated files and these appear automatically.
I guess we have some changes in a previous commit or in upstream repo somebody added something but forgot to update the auto-generated files.
But since these are auto-generated so I think we should leave them untouched.